MultiplayerMod/Game/Building/BuildValidation.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using HarmonyLib;
+
+namespace MultiplayerMod.Game.Building;
+
+[HarmonyPatch]
+public static class BuildValidation {
+
+    public static bool Enabled { get; set; } = true;
```

Comment:
It is quite hard to follow (for me personally). 
What do you think calling it something like `PatchEnabled` (and invert value as I get it)

Reply:
Don't assume it's a patch, yet it contains patch instructions. API should be readable for maintainable code, so it's `BuildValidation.Enabled = true | false` and literally means enable or disable validation during build process.
Nevertheless it will probably be replaced with game context operations later (it's still a draft PR, mostly for me to track changes).

Reply:
Replaced with `DisableBuildingValidation` context.